apps/web/src/components/SEBoundary/SEBoundary.tsx

```diff
@@ -0,0 +1,24 @@
+import type { ComponentProps } from 'react';
+import { AsyncBoundary } from '@toss/async-boundary';
+
+const INIT_PENDING_FALLBACK = <></>;
+const INIT_ERROR_FALLBACK = () => null;
+
+type BaseAsyncBoundaryProps = ComponentProps<typeof AsyncBoundary>;
+
+interface SEBoundaryProps extends Omit<BaseAsyncBoundaryProps, 'pendingFallback' | 'rejectedFallback'> {
+  pendingFallback?: BaseAsyncBoundaryProps['pendingFallback'];
+  rejectedFallback?: BaseAsyncBoundaryProps['rejectedFallback'];
+}
+
+function SEBoundary({ pendingFallback, rejectedFallback, ...props }: SEBoundaryProps) {
```

Comment:
```suggestion
function SEBoundary({ pendingFallback = INIT_PENDING_FALLBACK, rejectedFallback = INIT_ERROR_FALLBACK, ...props }: SEBoundaryProps) {
```

요롷게 기본 값으로 넣어주면 어떨까요 ??

의미에 더 부합하는 문법이라 생각돼서요 🤔